docs/manual/service_catalog/docs.md

```diff
@@ -0,0 +1,35 @@
+# Docs
+
+Docs section allow administrators to create and link documentation to Squest services or operations.
+
+Documentation are writen with Markdown syntax.
+
+## Linked to services
+
+When linked to one or more service, the documentation is shown in each "instance detail" page that correspond to the type of selected services.
+
+Jinja templating is supported with the `instance` as context.
+
+E.g:
+```
+You instance is available at {{ instance.spec.dns }}
+```
+
+## Linked to operations
+
+When linked to one or more operation, the documentation is shown during the survey of the selected operations.
+
+Like for services, Jinja templating is supported with the `instance` as context.
+
+!!!note
+
+    No instance context is injected on "create" operations as the instance doesn't exist yet at this stage
```

Comment:
Not available too if you click directly from "Docs" (sidebar), you need to access it trough instance_detail to use jinja templating